frontend/src/metabase/entities/collections/constants.ts

```diff
@@ -3,7 +3,7 @@ import { t } from "ttag";
 export const DEFAULT_COLLECTION_COLOR_ALIAS = "brand";
 
 export const ROOT_COLLECTION = {
-  id: "root",
+  id: "root" as const,
```

Comment:
Curios why do we need that?

Reply:
because `CollectionId` is defined as `"root" | number`. By default "root" will be `string` and not `"root"`